packages/in_app_purchase/test/store_kit_wrappers/sk_product_test.dart

```diff
@@ -17,19 +17,10 @@ void main() {
         () {
       final SKProductSubscriptionPeriodWrapper wrapper =
           SKProductSubscriptionPeriodWrapper.fromJson(
-              buildSubscriptionPeriodMap(dummySubscription));
+              buildSubscriptionPeriodMap(dummySubscription)!);
       expect(wrapper, equals(dummySubscription));
     });
 
-    test(
-        'SKProductSubscriptionPeriodWrapper should have properties to be null if map is empty',
-        () {
-      final SKProductSubscriptionPeriodWrapper wrapper =
-          SKProductSubscriptionPeriodWrapper.fromJson(<String, dynamic>{});
-      expect(wrapper.numberOfUnits, null);
-      expect(wrapper.unit, null);
```

Comment:
For these removed tests, shouldn't we be able to assert that everything has expected default values if the map is empty, rather than removing the tests?